packages/web_ui/src/components/SlavesPage.jsx

```diff
@@ -1,28 +1,46 @@
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 
 import libLink from "@clusterio/lib/link";
 
 import DataTable from "./data-table";
 import ControlContext from "./ControlContext";
 import PageLayout from "./PageLayout";
-
+import { copySync } from "fs-extra";
```

Comment:
This is nonsense. It's not used, and in the web code `fs-extra` is an empty package.